dev/github/Dockerfile-jdk11

```diff
@@ -11,7 +11,22 @@
 
 # See https://hub.docker.com/r/alluxio/alluxio-maven for instructions on running the image.
 
+ARG RUST_VERSION=1.66.1
```

Comment:
one more comment: is there a link that you got these docker steps from? i prefer to leave a comment with the reference link so someone else trying to update this code in the future doesn't need to do the same research again

Reply:
sure